src/NodeServices/ExecutionSession.cs

```diff
@@ -81,6 +81,10 @@ public class ParameterKeys
         /// The return value is an ILogger
         /// </summary>
         public static readonly string Logger = nameof(Logger);
-        
+
+        /// <summary>
+        /// The path to the preference file that is being used by Dynamo.
+        /// </summary>
+        public static readonly string PreferenceFilePath = nameof(PreferenceFilePath);
```

Comment:
Hi @twastvedt I think @mjkkirschner was asking here if we want to make the entire `PathManager` available instead of just the `PreferenceFilePath`.

Reply:
Yes, I'm aware of what he's asking. I'm not sure how to answer. Are you?